README.md

```diff
@@ -10,8 +10,10 @@ Courtbot is a simple web service for subscribing to case hearing details via SMS
   - Install [asdf](https://github.com/asdf-vm/asdf#setup)
   - Add Elixir asdf plugin: [asdf-elixir](https://github.com/asdf-vm/asdf-elixir) 
   - Goto where you've cloned the repo and run: `asdf install`
+    - If you are on OSX and installed `openssl` with brew, run `ERLANG_OPENSSL_PATH="/usr/local/opt/openssl" asdf install` 
```

Comment:
I was reading the asdf-erlang bug tracker and you having this issue maybe due to having an outdated version of the erlang plugin. [This is the discussion](https://github.com/asdf-vm/asdf-erlang/issues/86). In general I'd prefer not to add specific platform setup steps to the readme. As these should be resolved in the tools we use to be cross-platform. If you don't think the issue above would solve your original issue then we can add it to the wiki.